examples/fork_children/main.rb

```diff
@@ -32,13 +33,13 @@
   # NOTE: disabling discards the buffer by default, so all spans
   # get cleared here except the final toggle span
   10.times do
-    LightStep.disable
-    LightStep.enable
-    LightStep.disable
-    LightStep.disable
-    LightStep.enable
-    LightStep.enable
-    span = LightStep.start_span("my_toggle_span-#{Process.pid}")
+    OpenTracing.global_tracer.disable
```

Comment:
nit: "morally" speaking I think it would make more sense to create a local var called `lightstep_tracer` and invoke disable/enable on it for clarity